src/Sarif/Comparers/ArtifactComparer.cs

```diff
@@ -1,114 +1,110 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+// Copyright (c) Microsoft.  All Rights Reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.CodeDom.Compiler;
 using System.Collections.Generic;
 
-/// <summary>
-/// Note: This comparer may not have all properties compared. Will be replaced by a comprehensive
-/// comparer generated by JSchema as part of EqualityComparer in a planned comprehensive solution.
-/// Tracking by issue: https://github.com/microsoft/jschema/issues/141
```

Comment:
Does this PR address this issue?

Reply:
Correct. Previously manually created some comparer classes for Sarif entities. With the new change in JSchema, comprehensive auto-generated comparer classes, those manually created ones can be removed.

also related issue: #2424

Reply:
Awesome. If you're removing a comment like this in the future, it would be helpful to add a comment in the PR to link it all together.
If this PR fixes the Issue #2424, you probably want to note that in the description.